File: round_test/round.java
public class Test {
    public static void main(String args[]){
        // ATOMIC FUNCTION:round
        // LANG:java
        // FUNCTION: Math.round(start_val)

        double start_val=1.2;
        double expected_end_val=1;
        double end_val=Math.round(start_val);

        // TEST
        if(end_val==expected_end_val){
            System.out.println("PASS");
        }
        else{
            System.out.println("FAIL");
        }

        
    }
}

File: round/round.php
<?php
// ATOMIC_FUNCTION:  round($val, $precision)
// TAGS: math，numeric
// LANG:  php
// FUNCTION: round($val, $precision)

$start_val = 1.2;
$expected_end_val = 1;
$precision = 0;
$end_val = round($start_val,$precision);

// TEST
if($end_val == $expected_end_val) {
	echo "PASS";
} else {
	echo "FAIL";
}

?>
